packages/web-components/src/components/callout-with-media/__stories__/callout-with-media.stories.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit-element';
+import { select } from '@storybook/addon-knobs';
+import styles from './callout-with-media.stories.scss';
+import '../callout-with-media';
+import '../callout-with-media-image';
+import '../callout-with-media-video';
+import textNullable from '../../../../.storybook/knob-text-nullable';
+import readme from './README.stories.mdx';
+
+const image = html`
+  <dds-callout-with-media-image
+    alt="Image alt text"
+    default-src="https://dummyimage.com/672x378/ee5396/161616&text=16:9"
```

Comment:
```suggestion
    default-src="https://dummyimage.com/672x378/ee5396/161616&amp;text=16:9"
```

Reply:
Thank you!!! And changing it to ready for review